views/account/change-password.ejs

```diff
@@ -0,0 +1,49 @@
+<% layout('layouts/boilerplate')%>
+
+<div class="main-box">
+  <%-include('../partials/account-navigation')%>
+
+  <div class="info-box">
+    <form
+      action="/update-password"
+      method="POST"
+      class="change-profile-form validated-form"
+      novalidate
+    >
+      <div class="profile-form-input-div">
```

Comment:
Would say, move these inputs to new line on mobile for better user experience + input fields will look nice, even and uniform.